requirements-optional.txt

```
@@ -1,4 +1,7 @@
 -r requirements.txt
+# Alibaba
+aliyun-python-sdk-alimt==3.2.0
+aliyun-python-sdk-core-v3==2.13.33
```

`aliyun-python-sdk-alimt` depends on `aliyun-python-sdk-core`, why is `aliyun-python-sdk-core-v3` needed?

Also I suggest using less strict dependency pin here, something like:

```suggestion
aliyun-python-sdk-alimt>=3.2.0,<4.0.0
```

---

Ok got it. `aliyun-python-sdk-core-v3` was mentioned in the docs, if using python > 3.0.